Example/Backpack.xcodeproj/project.pbxproj

```diff
@@ -41,6 +41,14 @@
 		6048F77D2028BCE000C53A8B /* BPKRadiiViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 6048F77C2028BCE000C53A8B /* BPKRadiiViewController.m */; };
 		6055821721523A1300BF9F3E /* IconsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6055821621523A1300BF9F3E /* IconsViewController.swift */; };
 		6055821921523FAD00BF9F3E /* IconsPreviewCollectionViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6055821821523FAD00BF9F3E /* IconsPreviewCollectionViewCell.swift */; };
+		606BBA5F260DE0C700683FBA /* SkyscannerRelativeiOS-BoldItalic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA57260DE0C700683FBA /* SkyscannerRelativeiOS-BoldItalic.ttf */; };
+		606BBA60260DE0C700683FBA /* SkyscannerRelativeiOS-Italic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA58260DE0C700683FBA /* SkyscannerRelativeiOS-Italic.ttf */; };
+		606BBA61260DE0C700683FBA /* SkyscannerRelativeiOS-Black.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA59260DE0C700683FBA /* SkyscannerRelativeiOS-Black.ttf */; };
+		606BBA62260DE0C700683FBA /* SkyscannerRelativeiOS-MediumItalic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA5A260DE0C700683FBA /* SkyscannerRelativeiOS-MediumItalic.ttf */; };
+		606BBA63260DE0C700683FBA /* SkyscannerRelativeiOS-BlackItalic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA5B260DE0C700683FBA /* SkyscannerRelativeiOS-BlackItalic.ttf */; };
+		606BBA64260DE0C700683FBA /* SkyscannerRelativeiOS-Medium.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA5C260DE0C700683FBA /* SkyscannerRelativeiOS-Medium.ttf */; };
+		606BBA65260DE0C700683FBA /* SkyscannerRelativeiOS-Book.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA5D260DE0C700683FBA /* SkyscannerRelativeiOS-Book.ttf */; };
+		606BBA66260DE0C700683FBA /* SkyscannerRelativeiOS-Bold.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 606BBA5E260DE0C700683FBA /* SkyscannerRelativeiOS-Bold.ttf */; };
```

Comment:
This shouldn't be checked in, otherwise the app can't be built without relative files

Reply:
Thanks, but this is a draft PR so hold your horses 😛